config/routes/support.rb

```diff
@@ -17,6 +17,7 @@
       resource :locations do
         resource :multiple, only: %i[new create], on: :member, controller: 'providers/locations/multiple' do
```

Comment:
can we wrap this block in a:

```
scope module: :providers do
end
```

then you don't have to specify `providers/*` in each route call (or move it to where it makes sense)

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/3420/commits/46ded56c6ddeab1ee86ceddb05e5010ce2e85c53